libs/sdk-core/src/binding.rs

```diff
@@ -261,6 +261,12 @@ pub fn payment_by_hash(hash: String) -> Result<Option<Payment>> {
         .map_err(anyhow::Error::new::<SdkError>)
 }
 
+/// See [BreezServices::set_payment_metadata]
```

Comment:
@hydra-yse we should also generate the dart bindings using the flutter rust bridge